README.md

```diff
@@ -116,6 +112,15 @@ Additional resources:
 * [serverfull](https://github.com/asecurityteam/serverfull)
 * [serverfull-gateway](https://github.com/asecurityteam/serverfull-gateway)
 
+### Gateway Environment Variables
```

Comment:
task: If you add a new section like this then you need to also regenerate the table of contents and page anchors or it won't show up correctly.

Reply:
done!